lib/order.rb

```diff
@@ -0,0 +1,47 @@
+class Order
+
+  attr_reader :basket
+
+  def initialize
+    @basket = []
+  end
+
+  def add_to_basket(menu, dish_number, quantity)
+    quantity.times do
+      @basket << menu.select(dish_number)
+    end
+  end
+
+  def summary
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for summary is too high. [17.06/15]